build-parent/pom.xml

```diff
@@ -52,6 +52,7 @@
     <version.io.quarkiverse.loggingjson.quarkiverse-logging-json>1.1.1</version.io.quarkiverse.loggingjson.quarkiverse-logging-json>
     <version.io.quarkiverse.operatorsdk>3.0.6</version.io.quarkiverse.operatorsdk>
     <version.io.github.classgraph>4.8.143</version.io.github.classgraph>
+    <version.com.fasterxml.jackson.datatype>2.13.1</version.com.fasterxml.jackson.datatype>
```

Comment:
We should enforce the version to be the same that Quarkus brings in with quarkus bom.
Can you please verify if it is not already available via quarkus bom? 
We don't declare the version of jackson in the rest of the platform so I expect this to be available via platform too

Reply:
Thank you. Good catch! It is already managed by Quarkus. :+1: